src/components/results/securityanalysis/security-analysis-result-tab.tsx

```diff
@@ -115,6 +204,22 @@ export const SecurityAnalysisResultTab: FunctionComponent<
         [securityAnalysisResult]
     );
 
+    const [filterEnumsLoading, filterEnums] = useFetchFiltersEnums(
+        result?.empty
+    );
+
+    useEffect(() => {
+        if (result?.totalElements) {
+            setCount(result.totalElements);
```

Comment:
in some cases the result elements are not all showed so the count does not match with the total rows like here 
`
export const flattenNmKResultsContingencies = (
    intl: IntlShape,
    result: ConstraintsFromContingencyItem[] = []
) => {
    const rows: SecurityAnalysisNmkTableRow[] = [];

    result?.forEach(
        ({
            subjectLimitViolations = [],
            contingency,
        }: ConstraintsFromContingencyItem) => {
            if (!!subjectLimitViolations.length) {
`

Reply:
if the totalElements is equal to 0 the count will not be updated

Reply:
for the first one, Kevin will work on the backend side to fixe this issue (because it have no sense to have result with subjectLimitViolations empty)

for the second one, I fixed this, thank you